kolibri/plugins/user_profile/assets/src/views/ChangeFacility/SelectFacility.vue

```diff
@@ -120,39 +117,73 @@
         return savedAddressesInitiallyFetched.value && discoveredAddressesInitiallyFetched.value;
       });
       const storageFacilityId = useLocalStorage('kolibri-lastSelectedFacilityId', '');
+
+      // data:
+      const availableAddressIds = ref([]);
```

Comment:
Are these intended to be persistent across multiple uses of this component?

Reply:
I am not sure of understanding your question: this is a data variable that's used several times inside the component, not only in the `setup`, but also inside some `watch` methods. It has to be a reactive variable.